intro-2018-06.md

```diff
@@ -0,0 +1,178 @@
+
+![left](assets/stickers - joshparnham_2018-Feb-08.jpg)
+
+# Melbourne CocoaHeads
+## June 14, 2018
+### Join slack @ slack.melbournecocoaheads.com
+### Twitter @melbournecocoa
+
+---
+
+![left](assets/stickers - joshparnham_2018-Feb-08.jpg)
+
+# Melbourne CocoaHeads
+## Kicking off @ 6:30pm
+### Join slack @ slack.melbournecocoaheads.com
+### Twitter @melbournecocoa
+### Stickers up front!
+---
+
+# Welcome
+# :wave:
+
+---
+
+# Welcome :wave:
+
+![right 60%](assets/CocoaHeads.png)
+
+- The Month That Was
+- "Who's this guy?"
+- Upcoming CocoaHeads events
+- Code of Conduct
+- Tonight's agenda & sponsors
+- Who's hiring
+
+---
+
+# The Month That Was
+## May 8 - June 14 - WWDC 2018
+
+---
+
+# WWDC 2018
+
+ - iOS 12 - (ML, AR, Siri Extensions, Performance++, Screen Time)
+ - MacOS Mojave - (Dark Mode, UIKit on macOS)
+ - Xcode 10 - (Testing improvements, Swift 4.2, feelsgoodman.gif)
+
+---
+
+# Hi, I'm Jesse
+
+![left](assets/jtanang_2015-Aug-13.jpg)
+
+- Developing for iOS since 2010
+- realestate.com.au, Beanhunter, RMIT
+- Melbourne CocoaHeads organiser since ~2014
+
+---
+
+# Upcoming Events
+
+![left](assets/highres_469612543.jpeg)
+
+- Hack Night - June 20th @ IBM
+- Drinks Night - June 26th
+- NS Breakfast - July 6th
+
+---
+
+# Finding CocoaHeads
+
+- melbournecocoaheads.com
+- twitter.com/@melbournecocoa
+- slack.melbournecocoaheads.com
+- melbournecocoaheads.com/live (YouTube)
+- jesse@melbournecocoaheads.com
+
+---
+
+# CocoaHeads on YouTube
+
+![inline fit](assets/youtube-june-2018.png)
+
+ - melbournecocoaheads.com/live
+
+---
+
+# Code of Conduct
+
+---
+
+> Our community is dedicated to providing an inclusive environment for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lack thereof), or technology choices.
+
+---
+
+## melbournecocoaheads.com/code-of-conduct
+
+---
+
+## codeofconduct@melbournecocoaheads.com
+
+^ All complaints made in any of these ways will remain confidential, be taken seriously, investigated, and dealt with appropriately.
+
+---
+
+# Sponsors for 2018
+
+---
+
+# Major Sponsor
+
+![inline](assets/sponsors/Realestate.png)
+
+---
+
+# Silver Sponsors
+
+![inline](assets/sponsors/IttyBittyApps.png)![inline](assets/sponsors/Bilue.png)
+![inline](assets/sponsors/JTribe.png)![inline](assets/sponsors/Cognizant.png)![inline](assets/sponsors/Xplor.png)
+
+---
+
+# 2018 Venue
+
+![inline 100%](assets/sponsors/YBF Ventures.png)
+
+---
+
+# Tonight's Agenda
+
+Time|Speaker|Topic|
+---|---|---
+6:00 - 6:30 | Arrival and :pizza:
+6:30 - 6:40 | Intro & Who's Hiring
+6:45 - 7:00 | Matt Delves | Mac OS Mojave Dark Mode
+7:00 - 7:15 | Vlas Voloshin | Marzipan's Internals
+7:10 - 7:30 | Break
+7:30 - 8:00 | Barry Scott | Structure and Interpretation of Code Coverage
```

Comment:
Might be worth adding the pub to the agenda at the start so people know it is on? It may change whether they decide to hang around or not.